sk-csharp-azure-functions/config/KernelSettings.cs

```diff
@@ -7,6 +7,9 @@ internal class KernelSettings
 {
     public const string DefaultConfigFile = "config/appsettings.json";
 
+    [JsonPropertyName("endpointType")]
+    public string EndpointType { get; set; } = EndpointTypes.TextCompletion;
```

Comment:
If we make it as `ChatCompletion` by default, will it reduce error probability? If yes, then I think it worth to change it.